src/topology/algebra/group.lean

```diff
@@ -460,6 +461,14 @@ instance pi.topological_group {C : β → Type*} [∀ b, topological_space (C b)
   [∀ b, group (C b)] [∀ b, topological_group (C b)] : topological_group (Π b, C b) :=
 { continuous_inv := continuous_pi (λ i, (continuous_apply i).inv) }
 
+open mul_opposite
+
+/-- If multiplication is continuous in `α`, then it also is in `αᵐᵒᵖ`. -/
+@[to_additive "If addition is continuous in `α`, then it also is in `αᵃᵒᵖ`."]
+instance [group α] [topological_group α] :
+  topological_group αᵐᵒᵖ :=
+{ continuous_inv := continuous_induced_rng $ (@continuous_inv α _ _ _).comp continuous_unop }
```

Comment:
Pleas add the corresponding `has_continuous_something` and `topological_ring` instances.

Reply:
added now!